src/carbonio-files-ui-common/views/components/NodeListItem.tsx

```diff
@@ -221,7 +221,7 @@ export const NodeListItem = ({
 			} else if (includes(permittedContextualMenuActions, Action.Edit)) {
 				// if node can be opened with docs on edit mode, open editor
 				openNodeWithDocs(node.id);
-			} else if ($isSupportedByPreview) {
+			} else if ($isSupportedByPreview || (isFile(node) && node.type === NodeType.Video)) {
```

Comment:
The fact that we have to replicate this conditional 3 times makes me think we should either include it in the already existing function, or create a new util, which should return if a node can be initialized in ui-preview.
If we want to keep the concept of ui-preview distict from the concept of preview-service, we could keep 2 different functions, but I think here we should only be aware of the ui-preview one, hiding the preview-service concept behind it.
Of course, I'm open to discussion :nerd_face: